sites/bizbash/server/styles/index.scss

```diff
@@ -439,6 +439,17 @@ body {
     }
   }
 }
+.directory-facets {
+  & .input-wrapper {
```

Comment:
This doesn't follow BEM. The venue capacity field should either be a modifier, or it's own element or block